contrib/babelfishpg_unit/test_money.c

```diff
@@ -0,0 +1,298 @@
+#include "babelfishpg_unit.h"
+#include "../babelfishpg_money/fixeddecimal.c"
```

Comment:
why are we including C file here?

Reply:
Because babelfishpg_money has no header file like fixeddecimal.h 
So, we are directly including C file here